src/test/java/com/wikia/webdriver/common/core/helpers/FileTextLoader.java

```diff
@@ -0,0 +1,33 @@
+package com.wikia.webdriver.common.core.helpers;
+
+
+import com.wikia.webdriver.common.contentpatterns.PageContent;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class FileTextLoader {
+
+  public String loadFileTextContent(String filename) {
+    StringBuilder textContent = new StringBuilder();
+    String NL = System.getProperty("line.separator");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00117)